recipes/bridger/patches/rsem_sam.patch

```diff
@@ -0,0 +1,25 @@
+--- plugins/rsem/sam/Makefile	2017-01-24 08:02:17.000000000 +1000
++++ plugins/rsem/sam/Makefile.new	2017-01-24 08:08:35.000000000 +1000
```

Comment:
That's the whole point. You want to patch the file and have the result overwrite the original. Otherwise, the patched file will get ignored.